ydb/tests/library/harness/kikimr_config.py

```diff
@@ -432,7 +432,32 @@ def __init__(
 
             self.yaml_config["feature_flags"]["enable_external_data_sources"] = True
             self.yaml_config["feature_flags"]["enable_script_execution_operations"] = True
-
+            
+        if os.getenv("YAML_CONFIG") is not None:      
+            with open("/ydb_data/" + os.getenv("YAML_CONFIG")) as fh:
```

Comment:
use os.path.join instead of strings concatenation. In the case it will allow set full path instead of relative path/filename.

In other cases special function may add security, for example escape special symbols. Strings concatenation usually is last way.

Reply:
fixed